### PR TITLE
Leave reporting unmapped declared license issues up to the evaluator rules

### PR DESCRIPTION
**Pull request by fviernau**
See the individual commit messages for the details.

### CODE REVIEW

File: docs/examples/rules.kts [hunk 1]
```diff
@@ -121,6 +121,20 @@ val ruleSet = ruleSet(ortResult, packageConfigurationProvider) {
         }
     }
 
+    packageRule("UNMAPPED_DECLARED_LICENSE") {
+        require {
+            -isExcluded()
+        }
+
+        pkg.declaredLicensesProcessed.unmapped.forEach { unmappedLicense ->
+            error(
```
Comment (sschuberth) [inline docs/examples/rules.kts:130]: Shouldn't we stick to a warning here to have a 1:1 mapping to the previous code?

Comment (fviernau) [reply docs/examples/rules.kts]: Done.

**Comment by sschuberth** [comment]
BTW, this still is a change in behavior as the example rules are not applied by default. Which leads me to a related topic that I wanted to address earlier: As we have no default rules, running the evaluator with the standard configuration effectively does nothing, which might be confusing for new users. So I wonder whether we actually *should* have default rules (built-in as a resource into the evaluator), and whether these rules simply should be the example rules.

Any thoughts on that @fviernau and @tsteenbe?

**Comment by fviernau** [comment]
> BTW, this still is a change in behavior as the example rules are not applied by default. Which leads me to a related topic that I wanted to address earlier: As we have no default rules, running the evaluator with the standard configuration effectively does nothing, which might be confusing for new users. So I wonder whether we actually _should_ have default rules (built-in as a resource into the evaluator), and whether these rules simply should be the example rules.
> 
> Any thoughts on that @fviernau and @tsteenbe?

I think that's a good idea!

**Pull request merged**